src/operun/crm/browser/templates/account.pt

```diff
@@ -70,9 +70,46 @@
         </div>
 
         <h2 i18n:translate="account_view_notes_heading">Notes</h2>
-
         <div tal:condition="context/text" tal:replace="structure context/text/output"></div>
 
+        <div tal:define="attachments python:view.get_attachments()">
+          <h2 i18n:translate="account_view_attachments_heading">Attachments</h2>
+          <h3>Invoices</h3>
+          <hr>
+
+          <div class="invoice-item-wrapper" tal:repeat="invoice python:attachments['invoices']">
+            <ul>
+              <li tal:define="obj python:invoice.getObject();
+                               url obj/absolute_url;
+                               filename obj/file/filename">
+                <h4>
```

Comment:
Please don't use headings inside a list element. Just drop it and leave it as a default li.